core/src/main/scala/cats/collections/Dequeue.scala

```diff
@@ -87,11 +91,32 @@ sealed abstract class Dequeue[A] {
     */
   def :+(a: A): Dequeue[A] = snoc(a)
 
-  /**
-    * convert this queue to a stream of elements from front to back
-    */
-  @deprecated("Streaming is obsolete. Use either fs2, Monix, or iteratees.", "cats-collections 0.7.0")
-  def toStreaming: Streaming[A] = Streaming.unfold(this)(_.uncons)
+  def toIterator: Iterator[A] = new Iterator[A] {
+    private var pos: Dequeue[A] = Dequeue.this
+
+    override def hasNext: Boolean = !pos.isEmpty
+
+    override def next(): A = pos.uncons match {
+      case None => throw new NoSuchElementException()
+      case Some((a, rest)) =>
+        pos = rest
+        a
+    }
+  }
+
+  def to[Col[_]](implicit cbf: CanBuildFrom[Nothing, A, Col[A]]): Col[A] = {
+    val builder = cbf()
+    @tailrec def go(cur: Dequeue[A]): Unit = cur.uncons match {
+      case Some((a, rest)) =>
+        builder += a
+        go(rest)
+      case _ =>
```

Comment:
should this have an explicit `()`?

Reply:
No, I don't think so. Like an `if` without an `else`, this is implicitly unit.